Formula/g/git-cola.rb

```diff
@@ -16,6 +16,10 @@ class GitCola < Formula
   depends_on "pyqt"
   depends_on "python@3.14"
 
+  on_linux do
+    depends_on "qtwayland"
```

Comment:
this is something we do these days :) 

```suggestion
    depends_on "qtwayland" => :no_linkage
```